website/README.md

```diff
@@ -123,13 +119,51 @@ There are several custom markdown plugins that are available by default that enh
 
 A number of custom [mdx components](https://mdxjs.com/) are available for use within any `.mdx` file. Each one is documented below:
 
+#### Inline Alerts
+
+There are custom MDX components available to author alert data. [See the full documentation here](https://developer.hashicorp.com/swingset/components/mdxinlinealert). They render as colored boxes to draw the user's attention to some type of aside.
+
+```mdx
+## Alert types
+
+### Tip
+
+<Tip>
+  To provide general information to the user regarding the current context or
+  relevant actions.
+</Tip>
+
+### Highlight
+
+<Highlight>
+  To provide general or promotional information to the user prominently.
+</Highlight>
+
+### Note
+
+<Note>
+  To help users avoid an issue. Provide guidance and actions if possible.
+</Note>
+
+### Warning
+
+<Warning>
+  To indicate critical issues that need immediate action or help users
+  understand something critical.
+</Warning>
+
+## Title override prop
+
+<Note title="Hashiconf 2027">To provide general information.</Note>
+```
+
 #### Tabs
 
 The `Tabs` component creates tabbed content of any type, but is often used for code examples given in different languages. Here's an example of how it looks from the Vagrant documentation website:
 
 ![Tabs Component](https://p176.p0.n0.cdn.getcloudapp.com/items/WnubALZ4/Screen%20Recording%202020-06-11%20at%2006.03%20PM.gif?v=1de81ea720a8cc8ade83ca64fb0b9edd)
 
-> Please refer to the [Swingset](https://react-components.vercel.app/?component=Tabs) documentation for the latest examples and API reference.
+> Please refer to the [Swingset](https://react-components.vercel.app/?component=Tabs) documention for the latest examples and API reference.
```

Comment:
oooo looks like a typo, I'll fix in the package.

Reply:
fixed in https://github.com/hashicorp/web-platform-packages/pull/152/commits/aba2e2746b1c567a57a32988750452a8bba8de3e